peripherals/pwr/pwr_wl.yaml

```
@@ -0,0 +1,242 @@
+PWR:
+  CR1:
+    LPR:
+      MAIN_MODE: [0, "Voltage regulator in Main mode in Low-power run mode"]
+      LOW_POWER_MODE: [1, "Voltage regulator in low-power mode in Low-power run mode"]
+    VOS:
+      V1_2: [1, "1.2 V (range 1)"]
+      V1_0: [2, "1.0 V (range 2)"]
+    DBP:
+      Disabled: [0, "Access to RTC and backup registers disabled"]
+      Enabled: [1, "Access to RTC and backup registers enabled"]
+    FPDS:
+      Idle: [0, "Flash memory in Idle mode when system is in LPSleep mode"]
+      Power_down: [1, "Flash memory in Power-down mode when system is in LPSleep mode"]
+    FPDR:
+      Idle: [0, "Flash memory in Idle mode when system is in LPRun mode"]
+      Power_down: [1, "Flash memory in Power-down mode when system is in LPRun mode"]
+    SUBGHZSPINSSSEL:
+      SUBGHZSPICR: [0, "sub-GHz SPI NSS signal driven from PWR_SUBGHZSPICR.NSS (RFBUSYMS functionality enabled)"]
+      LPTIM3: [1, "sub-GHz SPI NSS signal driven from LPTIM3_OUT (RFBUSYMS functionality disabled)"]
+    LPMS:
+      Stop_0: [0, "Stop 0 mode"]
+      Stop_1: [1, "Stop 1 mode"]
+      Stop_2: [2, "Stop 2 mode"]
+      Standby: [3, "Standby mode"]
+      Shutdown: [4, "Shutdown mode"]
+  CR2:
+    PVME3:
+      Disabled: [0, "PVM3 (VDDA monitoring versus 1.62 V threshold) disable"]
+      Enabled: [1, "PVM3 (VDDA monitoring versus 1.62 V threshold) enable"]
+    PLS:
+      V2_0: [0, "2.0V"]
+      V2_2: [1, "2.2V"]
+      V2_4: [2, "2.4V"]
+      V2_5: [3, "2.5V"]
+      V2_6: [4, "2.6V"]
+      V2_7: [5, "2.7V"]
+      V2_8: [6, "2.8V"]
+      V2_9: [7, "2.9V"]
+      External: [8, "External input analog voltage PVD_IN (compared internally to VREFINT)"]
```

There is no 2.7V, and 2.8/2.9/external are off-by-one.
```suggestion
      V2_0: [0, "2.0V"]
      V2_2: [1, "2.2V"]
      V2_4: [2, "2.4V"]
      V2_5: [3, "2.5V"]
      V2_6: [4, "2.6V"]
      V2_8: [5, "2.8V"]
      V2_9: [6, "2.9V"]
      External: [7, "External input analog voltage PVD_IN (compared internally to VREFINT)"]
```